src/util.js

```diff
@@ -116,14 +117,15 @@ exports.deserialize = (data, callback) => {
 /**
  * Get the CID of the DAG-Node.
```

Comment:
Perhaps it should say `Get the CID of the CBOR node`?

Reply:
How about `Get the CID of the serialised CBOR node`?